physlr/physlr.py

```diff
@@ -998,10 +1002,26 @@ def physlr_subgraphs(self):
             os.makedirs(self.args.output)
         for u in progress(vertices):
             vertices_u = set()
+            u_mol = set()
+            if u in g:
+                u_mol.add(u)
+            else:
+                for i in range(100):
```

Comment:
100 is another 'magic number'? Is this a hack to catch all the molecules?
I know @jowong4  and I have discussed other approaches to avoid doing this in the past with his code

Reply:
Yes I am trying to include all the molecules; I can alternatively continue this loop infinitely until visiting molecule which is not present in the graph.. hum? 
@jowong4 Do you remember the solution you used for the same issue before?

Reply:
If I recall correctly (Correct me if I'm wrong Johnathan), we had an initial step to count the number of molecules per barcode, so you could stop once you'd seen all the molecule of a barcode

Reply:
Well yes, that was the idea but it doesn't actually work, because `xxx_1_0` doesn't have to exist for `xxx_1_1` to exist. The c++ code doesn't have this problem but I'll have to look into it again.

Reply:
So we never fixed that issue with the double molecule IDs? (ie. re-numbering if needed so avoid the double molecule nums) Sorry - my memory is fuzzy

Reply:
That said, if you can guarantee there is only one underscore, then the method would work, which I believe is the case here.

Reply:
> ble molecule IDs? (ie. re-numbering if needed so avoid the double molecule nums

not yet

Reply:
So if I continue the loop until it sees a barcode that is not present in the graph (assuming a single underscore), that does the same job. Its like I am counting the number of molecules but by checking them in the graph.

Reply:
```
        for u in progress(vertices):
            vertices_u = set()
            u_mol = set()
            if u in g:
                u_mol.add(u)
            else:
                mol = 0
                to_mol = u + "_" + str(mol)
                while to_mol in g:
                    u_mol.add(to_mol)
                    mol = mol + 1
                    to_mol = u + "_" + str(mol)```

Reply:
changes are made.